lib/transaction_builder.js

```diff
@@ -220,6 +220,49 @@ module.exports = class TransactionBuilder {
     return this
   }
 
+  /**
+  * Set the transaction builder with Previous Publickey and Previous Signature
+  * @param {String} to Previous Signature (hexadecimal)
+  * @param {String} to Previous PublicKey (hexadecimal)
+  */
+
+  setPreviousSignatureAndPreviousPublicKey(prevSign, prevPubKey) {
+    if (typeof (prevSign) == "string") {
```

Comment:
Can you do the check about the string or Uint8Array, then decode the hex if string

Reply:
I have added the additional checks.